data/sql/updates/pending_db_world/rev_1620674399849229322.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1620674399849229322');
+UPDATE `item_template` SET `minMoneyLoot`=50, `maxMoneyLoot`=100 WHERE `entry`=20708; -- Tightly Sealed Trunk
+UPDATE `item_template` SET `minMoneyLoot`=100, `maxMoneyLoot`=200 WHERE `entry`=21113; -- Watertight Trunk
+UPDATE `item_template` SET `minMoneyLoot`=100, `maxMoneyLoot`=200 WHERE `entry`=21150; -- Iron Bound Trunk
+UPDATE `item_template` SET `minMoneyLoot`=100, `maxMoneyLoot`=200 WHERE `entry`=21228; -- Mithril Bound Trunk
```

Comment:
```suggestion
UPDATE `item_template` SET `minMoneyLoot`=100, `maxMoneyLoot`=200 WHERE `entry` IN (
21113, -- Watertight Trunk
21150, -- Iron Bound Trunk
21228); -- Mithril Bound Trunk
```

Reply:
I wonder how long it will take until this optimization saved more cpu cycles than pushing the change through CI did cost. 😼